org.bitcoincore.bitcoin-qt.json

```
@@ -0,0 +1,87 @@
+{
+    "id": "org.bitcoincore.bitcoin-qt",
+    "runtime": "org.freedesktop.Platform",
+    "sdk": "org.freedesktop.Sdk",
+    "runtime-version": "18.08",
+    "command": "bitcoin-qt",
+    "finish-args": [
+	/* Wayland and Xorg support */
+	"--socket=x11",
+	"--socket=wayland",
+	/* Sound */
+	"--socket=pulseaudio",
+	/* Persistent storage */
+	"--persist=.bitcoin",
+	/* OpenGL access. */
+	"--device=dri",
+	/* IPC access. */
+	"--share=ipc",
+	/* Network acess. */
+	"--share=network"
+    ],
+    "modules": [
+	{
+	    "name": "bitcoin-core",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"install -m 0744 -D -t /app/bin bin/bitcoind",
+		"install -m 0744 -D -t /app/bin bin/bitcoin-qt",
+		"install -m 0744 -D -t /app/bin bin/bitcoin-cli",
+		"install -m 0744 -D -t /app/bin bin/test_bitcoin"
+	    ],
+	    "post-install": [
+		"/app/bin/test_bitcoin"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "only-arches": ["x86_64"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-x86_64-linux-gnu.tar.gz",
+		    "sha256": "5146ac5310133fbb01439666131588006543ab5364435b748ddfc95a8cb8d63f"
+		},
+		{
+		    "type": "archive",
+		    "only-arches": ["i386"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-i686-pc-linux-gnu.tar.gz",
+		    "sha256": "36ce9ffb375f6ee280df5a86e61038e3c475ab9dee34f6f89ea82b65a264183b"
+		},
+		{
+		    "type": "archive",
+		    "only-arches": ["arm"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-arm-linux-gnueabihf.tar.gz",
+		    "sha256": "3d7eb57290b2f14c495a24ecbab8100b35861f0c81bc10d86e5c0a8ec8284b27"
+		},
+		{
+		    "type": "archive",
+		    "only-arches": ["aarch64"],
+		    "url": "https://bitcoincore.org/bin/bitcoin-core-0.18.0/bitcoin-0.18.0-aarch64-linux-gnu.tar.gz",
+		    "sha256": "bfc3b8fddbb7ab9b532c9866859fc507ec959bdb82954966f54c8ebf8c7bb53b"
+		}
+
+	    ]
+	},
+	{
+	    "name": "misc",
+	    "buildsystem": "simple",
+	    "build-commands": [
+		"install -Dp -m 644 org.bitcoincore.bitcoin-qt.desktop /app/share/applications/org.bitcoincore.bitcoin-qt.desktop",
+		"install -Dp -m 644 org.bitcoincore.bitcoin-qt.svg /app/share/icons/hicolor/scalable/apps/org.bitcoincore.bitcoin-qt.svg",
+		"install -Dp -m 644 org.bitcoincore.bitcoin-qt.appdata.xml /app/share/appdata/org.bitcoincore.bitcoin-qt.appdata.xml"
```

```suggestion
		"install -Dp -m 644 org.bitcoincore.bitcoin-qt.appdata.xml /app/share/metainfo/org.bitcoincore.bitcoin-qt.appdata.xml"
```
appdata is the old folder to install the appdata files into.